src/R/Editor/Application.Test/Completion/IntellisenseTest.cs

```diff
@@ -197,6 +198,41 @@ public class IntellisenseTest : FunctionIndexBasedTest {
         }
 
         [Test]
+        public async Task R_CompletionFilesWorkingDirectory() {
+            using (var script = await _editorHost.StartScript(ExportProvider, RContentTypeDefinition.ContentType, Workflow.RSessions)) {
+                var myDocs = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
+                var folder = "_rtvs_test2_";
+
+                var testFolder = Path.Combine(myDocs, folder);
```

Comment:
I meant using `TestFilesFixture`. In this case, you don't have to erase the directory at the end of the test.